saleor/graphql/core/descriptions.py

```diff
@@ -0,0 +1 @@
+DEPRECATED_IN_3X = "DEPRECATED: this field will be removed in Saleor 4.0."
```

Comment:
At the meeting, we decided to add a new line at the start and end of this message.